readme.md

```diff
@@ -51,24 +51,49 @@ You can pass options to `awesome-typescript-loader` as a first argument
 // next.config.js
 const withAwesomeTypescript = require("next-awesome-typescript");
 
-const options = {
+const awesomeTypescriptOptions = {
   useCheckerPlugin: true,
   loaderOptions: {
     transpileOnly: false,
   },
 };
 
-module.exports = withAwesomeTypescript(options);
+module.exports = withAwesomeTypescript(awesomeTypescriptOptions);
 ```
 
-Optionally you can add your custom Next.js configuration as second parameter
+Optionally you can add your custom Next.js configuration as a parameter
```

Comment:
Maybe we'll merge last two cases into one?

Reply:
yes, If you want!